core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1316,6 +1316,20 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_TRANSPORT_REQUEST_TIMEOUT_MS =
```

Comment:
`MASTER_EMBEDDED_JOURNAL_TRANSPORT_REQUEST_TIMEOUT` rather than `MASTER_EMBEDDED_JOURNAL_TRANSPORT_REQUEST_TIMEOUT_MS` please. 
In 2.0 code base, we no longer use suffix `_MS` to indicate the unit